benchmarks/OpenTelemetrySdkBenchmarks.cs

```diff
@@ -17,7 +17,7 @@
 using BenchmarkDotNet.Attributes;
 using Benchmarks.Tracing;
 using OpenTelemetry.Trace;
-using OpenTelemetry.Trace.Configuration;
+using OpenTelemetry.Trace;
```

Comment:
This seems to be duplicated with line 19?

Reply:
yes. and build was showing it. fixed now